infra/main.bicep

```diff
@@ -47,101 +11,85 @@ param location string
 
 param resourceGroupName string = ''
 
-var abbrs = loadJsonContent('abbreviations.json')
-var uniqueSuffix = substring(uniqueString(subscription().id, resourceGroup.id), 1, 3) 
-
 param tags object
 
-//UAMI Module Parameters
-param msiName string = ''
-
 //OpenAI Module Parameters
 param openaiName string = ''
-@allowed(['gpt-35-turbo'])
-param gptModel string
-@allowed(['0301'])
-param gptVersion string
-param deployDalle3 bool = false
 
 //SQL Module Parameters
-@description('Deploy SQL Database? (required for SQL Plugin demo)')
-param deploySQL bool = true
 param sqlServerName string = ''
 param sqlDatabaseName string = ''
 @description('Set the administrator login for the SQL Server')
-@secure()
 param administratorLogin string
-@description('Set the administrator login password for the SQL Server')
 @secure()
+@minLength(8)
+@description('Set the administrator login password for the SQL Server')
 param administratorLoginPassword string
 
 //Speech Module Parameters
 param speechServiceName string = ''
-@description('Deploy Azure AI Speech service? (required for Text to Speech and Speech to Text Plugin demo)')
-param deploySpeechService bool = true
 
-@allowed(['Enabled', 'Disabled'])
-param publicNetworkAccess string = 'Enabled'
+var abbrs = loadJsonContent('abbreviations.json')
+var uniqueSuffix = substring(uniqueString(subscription().id, environmentName), 1, 5)
+
+var names = {
+    resourceGroupName: !empty(resourceGroupName) ? resourceGroupName : '${abbrs.resourcesResourceGroups}${uniqueSuffix}'
```

Comment:
not sure what happened, but azd up is generating rg-BLAH and not rg-env

Reply:
thats expected. I hashed the environmentName in line 33 and use that in the resource group name. I can update so that the resource group can be rg-<envName>, no strong opinion about that

Reply:
it should be rg-env, it was rg-BLASHDF.